README.md

```diff
@@ -1,44 +1,61 @@
-# Introducing Steem (beta)
+# Steem - The Blockchain That Will Tokenize The Web
 
-Steem is a Delegated Proof of Stake blockchain that uses a "Proof of Brain" social consensus algorithm for token allocation.
+Welcome to the official repository for Steem, the blockchain that will revolutionize the web, and soon the blockchain for Smart Media Tokens!
 
-  - Currency symbol STEEM.
-  - 10% APR inflation narrowing to 1% APR over 20 years.
-  - 75% of inflation to "Proof of Brain" social consensus algorithm.
-  - 15% of inflation to stake holders.
-  - 10% of inflation to block producers.
 
-# Public Announcement & Discussion
 
-Steem was announced on the
-[Bitcointalk forum](https://bitcointalk.org/index.php?topic=1410943.new) prior to
-the start of any mining.
+Steem is the first blockchain which introduced the "Proof of Brain" social consensus algorithm for token allocation.
 
-# No Support & No Warranty
+Being one of the most actively developed blockchain projects currently in existence, it's become fertile soil for entrepreneurial pursuits. It has also become home for many cryptocurrency centric projects.
 
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
-IN THE SOFTWARE.
+Steem aims to be the preferred blockchain for dApp development with Smart Media Tokens at its core. With SMTs, everyone can leverage the power of Steem.
+
+Originally, Steem was announced on the [Bitcointalk forum](https://bitcointalk.org/index.php?topic=1410943.new) prior to the start of any mining. (Steem is currently not mineable)
+
+## Documents
+
+* Whitepaper: [steem.io/SteemWhitepaper.pdf](https://steem.io/SteemWhitePaper.pdf)
+* Bluepaper: [steem.io/SteemBluepaper.pdf](https://steem.io/steem-bluepaper.pdf)
```

Comment:
The actual link has a dash in it, and works. However the displayed link is a 404, due to lacking a dash.